### ISSUE TITLE
Deleted C++ object: BrowserPage in DownloadManager.fetch_request

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#1241]
- `:later 1000 :tab-close`
- Start download
- Answer filename question when tab is closed

```
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/qutebrowser/browser/downloads.py", line 742, in <lambda>
    **kwargs))
  File "/usr/lib/python3.5/site-packages/qutebrowser/browser/downloads.py", line 760, in fetch_request
    nam = page.networkAccessManager()
RuntimeError: wrapped C/C++ object of type BrowserPage has been deleted
```

This also happened in 2 reports, probably due a slow connection and lots of downloading/closing tabs?

**Comment by amacfie** [comment]
Doesn't reproduce?

**Comment by The-Compiler** [comment]
Most likely fixed in c363982d057a82ef23edb4a506f06eedda732913.
